class.dynamic_cdn.php

```diff
@@ -43,14 +43,18 @@ public function init() {
 		$filter_priority = apply_filters( 'dynamic_cdn_filter_priority', 10 );
 		$action_priority = apply_filters( 'dynamic_cdn_action_priority', 10 );
 		
-		$this->uploads_only = apply_filters( 'dynamic_cdn_uploads_only', false );
+		if ( defined( 'DYNCDN_UPLOADS_ONLY' ) ){
```

Comment:
Talked about this in slack, adding it in here for reference. 

Even if we use the tenup plugin to hook we'll still need to allow for configuration via wp-config. I'd like to leave it here. That gives us the option of hooking or wp-config.